.github/workflows/verify_and_publish.yml

```diff
@@ -177,11 +177,80 @@ jobs:
         if-no-files-found: error
 
 
+  test_cli:
+    needs: format
+    runs-on: ubuntu-22.04
+
+    # This is needed for 'guestmount' / 'libguestfs'
+    env:
+      LIBGUESTFS_BACKEND: direct
+
+    container:
+      image: ghcr.io/refenv/cijoe-docker:latest
+      options: >
+        --device=/dev/kvm
+        --device=/dev/fuse
+        --cap-add=SYS_ADMIN
+        --security-opt apparmor=unconfined
+
+    strategy:
+      fail-fast: false
+      matrix:
+        cli:
+        - {name: "integrity-check", cmd: "$(which cijoe) example/cijoe-workflow.yaml -c example/cijoe-config.toml --integrity-check"}
```

Comment:
For readability, how about something like this form instead:

```
  - name: "integrity-check"
    cmd: "$(which cijoe) example/cijoe-workflow.yaml -c example/cijoe-config.toml --integrity-check"
  - name: "script-target"
    cmd: "$(which cijoe) core.example_script_default -c example/cijoe-config.toml"
  - name: "workflow-steps"
    cmd: "$(which cijoe) example/cijoe-workflow.yaml -c example/cijoe-config.toml script_with_args"}
  - name: "emit-example"
    cmd: "$(which cijoe) --example core.default"
  - name: "archive"
    cmd: "$(which cijoe) --archive"
```
?

Reply:
yes of course, i have fixed that now :)